ospfd/ospf_te.c

```diff
@@ -1244,7 +1244,10 @@ ospf_mpls_te_lsa_new (struct ospf_area *area, struct mpls_te_link *lp)
       tmp = SET_OPAQUE_LSID (OPAQUE_TYPE_INTER_AS_LSA, lp->instance);
       lsa_id.s_addr = htonl (tmp);
 
-      struct ospf *top = ospf_lookup ();
+      struct ospf *top = area->ospf;
```

Comment:
After more investigation, there is many call to ospf_lookup() function in ospf_te.c, ospf_ri.c and ospf_opaque.c to manage LSA Type 11 with AS flooding scope. All these instance must be replace with new VRF API. But, the problem is to get the vrf_id.

It needs deep investigation in OPAQUE LSA management to determine if the vrf_id could be accessible or need to be pass to these functions.

In this state, this PR could not be merge without breaking OPAQUE LSA Type 11 flooding.